pkg/util/proxyconfig/merge.go

```diff
@@ -19,43 +17,27 @@ import (
 // string of noProxy settings. If no user supplied noProxy settings are
 // provided, a comma-separated string of cluster-wide noProxy settings
 // are returned.
-func MergeUserSystemNoProxy(proxy *configv1.Proxy, infra *configv1.Infrastructure, network *configv1.Network, cluster *corev1.ConfigMap) (string, error) {
+func MergeUserSystemNoProxy(proxy *configv1.Proxy, infra *configv1.Infrastructure, network *configv1.Network,
```

Comment:
not usable by installer as you have the configmap to get install-config here https://github.com/openshift/cluster-network-operator/pull/296/files#diff-b9dde8483f6b2bbdb5b0a7695c57743aR40